src/platforms/alexa/CanFullfillintent.ts

```diff
@@ -0,0 +1,60 @@
+import { DialogState, Intent } from "ask-sdk-model";
```

Comment:
Since we didn't make any modification to the code I think we need to license this particular file as Apache, at least that's what I understand from a quick Google search. So instead of setting the MIT license at the top, we put the Apache license.